releases/24.0.0.5/full/Dockerfile.ubi.openjdk21

```diff
@@ -4,7 +4,7 @@ FROM $PARENT_IMAGE AS installBundle
 
 ARG VERBOSE=false
 ARG LIBERTY_VERSION=24.0.0.5
-ARG FEATURES_SHA={replace_with_correct_sha}
+ARG FEATURES_SHA=27d99efc0c30c8a26c6efda1838325f463030c43
```

Comment:
The value of FEATURES_SHA must be different from LIBERTY_SHA

It can be found pre-GA within the zip file `openliberty-MavenArtifact-*.zip` at 
`io/openliberty/features/features/<LIBERTY_VERSION>/features-<LIBERTY_VERSION>.json`